app/models/services/facebook.rb

```diff
@@ -24,14 +24,18 @@ def post_to_facebook(url, body)
 
   def create_post_params(post)
     message = post.message.plain_text_without_markdown
-    if post.photos.any?
-      message += " " + short_post_url(post, protocol: AppConfig.pod_uri.scheme,
-                                            host: AppConfig.pod_uri.authority)
-    end
-
-    {message: message,
-     access_token: access_token,
-     link:  post.message.urls.first
+    post_url = short_post_url(post, protocol: AppConfig.pod_uri.scheme, host: AppConfig.pod_uri.authority)
+
+    message = if message.empty? && post.photos.any?
+                post_url
+              else
+                "#{message} (via #{post_url})"
+              end
+
+    {
+      message: message,
+      access_token: access_token,
+      link: post.message.urls.first
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.

Reply:
I switched this to this format as requested but I can't get rid of this style error.  I'm at a loss on how to make this pass the style check except making it one line.  I'm following the example in the documentation for this Rocket/Colon style (table).  Can someone point out what I'm doing wrong here?  http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/AlignHash